website/questmaker/templates/constructor.html

```diff
@@ -2,53 +2,76 @@
 {% set active_page = active_page|default('constructor') -%}
 
 {% block content %}
-
 <script src="https://api-maps.yandex.ru/2.1/?apikey=57bad925-98eb-4a1e-8bf9-78a2f0a99e22&lang=ru_RU" type="text/javascript"></script>
     <main class="min-vh-100 min-vw-100 position-relative d-flex">
         <div class="position-absolute d-flex min-vw-100 min-vh-100">
-            <div class="col-1 position-fixed" style="box-sizing: border-box; padding-top: 88px; z-index: 100">
-                <div class="p-2">
-                    <button type="button" class="btn btn-primary" data-bs-toggle="modal" data-bs-target="#modal">
-                        Добавить блок
-                    </button>
-                    <button type="button" class="btn btn-primary mt-2" id="redactorQuest">
-                        Редактировать квест
-                    </button>
-                    <!--<button type="button" class="btn btn-primary" id="save">
-                        Сохранить
-                    </button>-->
-                </div>
-            </div>
+                            <!--  MENU begin -->
+                            
+                <aside class="bd-sidebar col-2 mt-4 gy-4 bg-light text-center" style="z-index: 1">
+                            <div class="mt-5">
+                                <button class="accordion-button flex align-items-center collapsed" data-bs-toggle="collapse" data-bs-target="#getting-started-collapse" aria-expanded="false">
```

Comment:
Where did you see the 'flex' class? And if don't know display: flex was adding by 'according-button'. Can you use the console?